ipywidgets/widgets/tests/test_send_state.py

```diff
@@ -23,3 +23,11 @@ def test_empty_hold_sync():
     with w.hold_sync():
         pass
     assert w.comm.messages == []
+
+
+def test_control():
+    comm = DummyComm()
+    Widget.close_all()
+    w = SimpleWidget()
+    Widget.handle_comm_opened_control(comm, {})
+    assert comm.messages
```

Comment:
Ideally the tests would be inspecting the messages a little closer, e.g. check the buffer decoding etc.